test/test.js

```diff
@@ -224,6 +224,10 @@ describe("Test Screenshots", function() {
       return driver.wait(
         until.elementLocated(By.css(".preview-button-save"))
       );
+    }).then((saveButton) => {
+      return driver.wait(
+        until.elementIsVisible(saveButton)
```

Comment:
If this isn't enough to eliminate intermittent failures, there's an `element.isEnabled()` method which polls for a visible form element to not have the `disabled` attribute on it. Some discussion in [this SO answer](https://stackoverflow.com/questions/26241495/how-do-i-wait-for-an-element-to-be-clickable-in-webdriverjs). We'd have to start toggling the `disabled` attribute when frames are loaded, but it could be worth it.